ipn/ipnlocal/local.go

```diff
@@ -3959,9 +3939,16 @@ func (b *LocalBackend) LogoutSync(ctx context.Context) error {
 		return errors.New("no controlclient")
 	}
 
-	err := cc.Logout(ctx)
-	b.stateMachine()
-	return err
+	if err := cc.Logout(ctx); err != nil {
+		return err
+	}
+	b.mu.Lock()
+	if err := b.pm.DeleteProfile(profile.ID); err != nil {
```

Comment:
there's still a way to force delete the profile if the cc Logout fails right?

Reply:
I don't think there is today, we need to add one.